constantine/serialization/codecs_banderwagon.nim

```diff
@@ -65,14 +59,14 @@ func make_scalar_mod_order*(reduced_scalar: var Fr[Banderwagon], src: array[32,
   res = true
   return res
 
-func serialize*(dst: var array[32, byte], P: EC_Prj): CttCodecEccStatus =
+func serialize*(dst: var array[32, byte], P: ECP_TwEdwards_Aff[Fp[Banderwagon]]): CttCodecEccStatus =
```

Comment:
The impl so far has been heavily using projective coordinates and probably need a review to use affine coordinates were ossible.